test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -432,12 +432,27 @@ def forward(self, x):
         m = M().eval()
         qconfig = torch.quantization.default_qconfig
         mp = _quantize_dbr.prepare(m, {'': qconfig}, (torch.randn(1, 1, 1, 1),))
+
         self.assertTrue(
             mp._auto_quant_state.idx_to_seen_q_op_infos[0].fusion_info is not None)
         self.assertTrue(
             mp._auto_quant_state.idx_to_seen_q_op_infos[1].fusion_info is not None)
-        # TODO(future PR): use fusion results to insert observers
-        # TODO(future PR): use fusion results to replace function at inference
+
+        # verify that the add relu is not observed
+        self.assertTrue(
+            '1' not in mp._auto_quant_state.tensor_id_to_observer)
```

Comment:
is it OK to hardcode the index here?

Reply:
yeah, it's not ideal, not sure if writing a more robust util for this is worth the effort now (it may be later)